builtin.c

```diff
@@ -197,6 +197,55 @@ static OP *ck_builtin_func1(pTHX_ OP *entersubop, GV *namegv, SV *ckobj)
     return newUNOP(opcode, wantflags, argop);
 }
 
+XS(XS_builtin_indexed)
+{
+    dXSARGS;
+
+    switch(GIMME_V) {
+        case G_VOID:
+            Perl_ck_warner(aTHX_ packWARN(WARN_VOID),
+                "Useless use of %s in void context", "builtin::indexed");
+            XSRETURN(0);
+            break;
+        case G_SCALAR:
+            Perl_ck_warner(aTHX_ packWARN(WARN_SCALAR),
+                "Useless use of %s in scalar context", "builtin::indexed");
+            mXPUSHi(items);
```

Comment:
Shouldn't this return the length of the would-be-returned list, i.e. `items * 2`?

Reply:
Oooh, probably yes.

Re-pushed.

Reply:
Do we want a test for it as well?

Reply:
Huh, yes. Good call as it turns out it wasn't correctly implemented anyway. Now fixed, along with test and docs - 9666de1f11